server/grpc_service.go

```diff
@@ -1685,3 +1686,82 @@ func checkStream(streamCtx context.Context, cancel context.CancelFunc, done chan
 	}
 	<-done
 }
+
+const globalConfigPath = "/global/config/"
+
+func (s *Server) StoreGlobalConfig(ctx context.Context, request *pdpb.StoreGlobalConfigRequest) (*pdpb.StoreGlobalConfigResponse, error) {
+	for _, item := range request.Changes {
+		name := globalConfigPath + item.GetName()
+		value := item.GetValue()
+		_, err := s.client.Put(ctx, name, value)
```

Comment:
Do we need to use transaction API?

Reply:
only the leader could config the global config so we don't need

Reply:
What I mean is these configs should be changed at the same time to ensure atomicity?

Reply:
done